grazer/src/main/java/edu/asu/diging/grazer/core/fileupload/service/impl/FileUploadServiceImpl.java

```diff
@@ -0,0 +1,96 @@
+package edu.asu.diging.grazer.core.fileupload.service.impl;
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+
+import org.apache.commons.io.FilenameUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.commons.CommonsMultipartFile;
+
+import edu.asu.diging.grazer.core.domain.impl.FileImpl;
+import edu.asu.diging.grazer.core.domain.impl.FileTransformationImpl;
+import edu.asu.diging.grazer.core.fileupload.db.impl.FileUploadDatabaseConnection;
+import edu.asu.diging.grazer.core.fileupload.service.IFileUploadService;
+
+@Service
+@PropertySource(value = "classpath:config.properties")
+public class FileUploadServiceImpl implements IFileUploadService {
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+    
+    @Autowired
+    private FileUploadDatabaseConnection connection;
+    
+    @Autowired
+    private FileImpl file;
+    
+    @Autowired
+    private Environment env;
+    
+    public void uploadFiles(HashMap<String, byte[]> filesMap) {
```

Comment:
should that ever be called by another class than this one? if not, it doesn't need to be public